### PR TITLE
EDUCATOR-5301: teams workflows should be in waiting status

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** When a teams workflow is created, the status should be WAITING

JIRA: [EDUCATOR-5301](https://openedx.atlassian.net/browse/EDUCATOR-5301)

**What changed?**

- Leave the workflow alone, don't set the value to TEAMS after creation.
- Update tests.

**Developer Checklist**

- [x] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [x] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

- Create a two team ORAs in a course with multiple learners. I'll call the ORAs A and B.
- On master, create one or more submissions for ORA A. After you submit, the Staff Grade section should be Not Available, and the message at the top of the screen should be .... my devstack is kinda fighting with me right now, but it should be kinda confusing, and the message in the "my grade" section should kinda imply that you have more work you have to do.
- Check out this branch, wait for things to reload, and then create one or more submissions for ORA B. THe staff grade section should show up, and both the messages should be something more helpful (update when devstack comes back on)
- Load the instructor dashboard openassessment page. ORA A should have zero listed responses. ORA B should have some number of "total responses" and an equal number under the "Staff" column. If you have `FEATURES['ENABLE_XBLOCK_VIEW_ENDPOINT'] = True` in your devstack.py, you should be able to click the "staff" number and verify that the numbers you see match.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

File: openassessment/workflow/models.py [hunk 1]
```diff
@@ -731,9 +731,6 @@ def start_workflow(cls, team_submission_uuid):  # pylint: disable=arguments-diff
         team_assessment_api = team_staff_step.api()
         team_assessment_api.on_init(team_submission_uuid)
 
-        team_workflow.status = TeamAssessmentWorkflow.STATUS.teams
```
Comment (nsprenkle) [inline openassessment/workflow/models.py:734]: Is there anywhere else that `STATUS.teams` lives as a constant that we should be removing?

**Comment by nsprenkle** [comment]
My before/after from testing:

ORA A
ORA B

**Review by nsprenkle — APPROVED** [review]
Reminder to bump version but otherwise testing & review looks good :shipit:
